### ISSUE TITLE
clearwater_snmp_alarm_agent_restart always fails

### ISSUE THREAD

**Issue by mirw** [Metaswitch/clearwater-snmp-handlers#110]
The `clearwater-snmp-alarm-agent` package contains a script `/usr/share/clearwater/infrastructure/scripts/restart/clearwater_snmp_alarm_agent_restart`.  This is intended to restart the agent to pick up new configuration.

Unfortunately, it always fails with the error `stop: Unknown instance:`.

This seems to be because the agent is managed by init rather than an init.d script, but I'm not all that familiar with this code.

Marking as medium priority because this could mean we don't pick up configuration changes when we should.

**Comment by rkd-msw** [comment]
This is because it uses `service clearwater-snmp-alarm-agent stop` - because we're being monitored by upstart rather than monit, we don't get automatically restarted.

I'm making related changes under https://github.com/Metaswitch/clearwater-snmp-handlers/pull/109, so I'll correct this to `service clearwater-snmp-alarm-agent restart` as part of this.

**Comment by rkd-msw** [comment]
Fixed by #109:

```
$ sudo /usr/share/clearwater/infrastructure/scripts/restart/clearwater_snmp_alarm_agent_restart
clearwater-snmp-alarm-agent stop/waiting
clearwater-snmp-alarm-agent start/running, process 24798
```
